src/10_functions.py

```diff
@@ -1,6 +1,9 @@
 # Write a function is_even that will return true if the passed-in number is even.
 
 # YOUR CODE HERE
+def is_even(num):
+    if num % 2 == 0:
```

Comment:
replace `if` with `return`, and you've got a one-line function that does the same thing :smile_cat:

Reply:
cool thank you